src/libcfitsio.jl

```diff
@@ -353,8 +353,8 @@ for (a,T,S) in (("ffukys", :ASCIIString, :(Ptr{UInt8})),
                 ("ffukyl", :Bool,        :Cint),
                 ("ffukyj", :Integer,     :Int64))
     @eval begin
-        function fits_update_key(f::FITSFile, key::ASCIIString, value::$T,
-                                 comment::@compat(Union{ASCIIString, Ptr{Void}})=C_NULL)
+        function fits_update_key(f::FITSFile, key::Compat.ASCIIString, value::$T,
```

Comment:
`$T` is sometimes `ASCIIString`. Change `:ASCIIString` to `:(Compat.ASCIIString)` on L352.